src/log_scanner.py

```diff
@@ -578,22 +576,22 @@ def __draft_pack_search_quick(self):
                         # Remove any prefix (e.g. log timestamp)
                         start_offset = line.find("{\"CurrentModule\"")
                         self.draft_log.info(line)
-                        draft_data = json.loads(line[start_offset:])
-                        payload_data = json.loads(draft_data["Payload"])
-                        pack_data = payload_data["DraftPack"]
-                        draft_status = payload_data["DraftStatus"]
+                        entry_string = line[start_offset:]
```

Comment:
I think this is a problem as I found that a Quick Draft still had "Payload" in the card pack log.

Reply:
The changes were verified for quick, premier, and traditional drafts. See test_log_scanner for the test cases.